core/modules/layout/layout.admin.inc

```diff
@@ -384,10 +384,16 @@ function layout_settings_form($form, &$form_state, Layout $layout) {
     '#required' => TRUE,
     '#access' => !$layout->isDefault(),
   );
+  if ($layout->layout_template != NULL) {
+    $layout_template = $layout->layout_template;
+  }
+  else {
+    $layout_template = config_get('layout.layout.default', 'layout_template');
+  }
```

Comment:
Lets add a comment here to explain what we are doing and why:
```suggestion
  else {
    // If no layout template has been selected, then pre-select the template
    // used by the default layout, to avoid AJAX validation errors in the form.
    $layout_template = config_get('layout.layout.default', 'layout_template');
  }
```

(something like that anyway - feel free to reword)